client/app/bundles/course/group/pages/GroupShow/GroupTableCard.jsx

```diff
@@ -85,9 +131,23 @@ const GroupTableCard = ({ group }) => {
           {members.map((m, index) => (
             <TableRow key={m.id} style={styles.rowHeight}>
               <TableCell style={styles.rowHeight}>{index + 1}</TableCell>
-              <TableCell style={styles.rowHeight}>{m.name}</TableCell>
               <TableCell style={styles.rowHeight}>
-                {roles[m.groupRole]}
+                <div className="flex grow">
+                  {m.name}
+                  {m.isPhantom ? <GhostIcon /> : ''}
```

Comment:
For brevity, `{m.isPhantom && <GhostIcon /> }`

Reply:
okay